woothemes/woocommerce-rest-api-docs
# Add docs for `/products` endpoint new parameters

Adds documentation for the REST endpoint introduced in https://github.com/woocommerce/woocommerce/issues/50462

### Testing steps

1. Run the build script found in the root of this repo: `./build.sh`
2. Find the `build` directory created by the script, and the `index.html` file within. Load that file in your web browser.
3. Click on the `Product > List all products` item in the left-side menu. Read the text and the code examples and make sure it all makes sense and there aren't any typos.

⚠️ We should not merge this until 9.6 is released ⚠️
